jackson-modules/jackson-conversions/src/main/java/com/baeldung/jackson/date/EventWithLocalDate.java

```diff
@@ -0,0 +1,36 @@
+package com.baeldung.jackson.date;
+
+import com.fasterxml.jackson.annotation.JsonFormat;
+import com.fasterxml.jackson.databind.annotation.JsonDeserialize;
+import com.fasterxml.jackson.databind.annotation.JsonSerialize;
+import com.fasterxml.jackson.datatype.jsr310.deser.LocalDateDeserializer;
+import com.fasterxml.jackson.datatype.jsr310.ser.LocalDateSerializer;
+
+import java.time.LocalDate;
+
+
+public class EventWithLocalDate {
+    public String name;
+
+    @JsonDeserialize(using = LocalDateDeserializer.class)
+    @JsonSerialize(using = LocalDateSerializer.class)
+    @JsonFormat(shape = JsonFormat.Shape.STRING, pattern = "dd-MM-yyyy")
+    public LocalDate eventDate;
+
+    public EventWithLocalDate() {
+        super();
```

Comment:
this shouldn't be present - `super()` the base class is `Object` which doesn't have a special constructor to call.